lib/metrics.js

```diff
@@ -92,6 +93,24 @@ function openwhisk(metrics) {
     return metrics;
 }
 
+/**
+ * Container memory size information from the environment
+ *
+ * @param {Object} [metrics={}] Metrics object
+ * @returns {Object} Metrics object
+ */
+function containerMemory(metrics) {
```

Comment:
IMO it would be fine to not have a separate function for this but just add it to the `openwhisk()` function. Would save a couple of lines and the changes in `lib/newrelic.js`.

Reply:
we should probably rename the function then, maybe `defaultMetrics()` or `containerMetrics`?

Reply:
Would the function rename be a breaking change for the components using this?

Reply:
I don't think any of the dependents use the Metrics class but to be safe I just kept the name the same